src/commands/deploy/genezio.ts

```diff
@@ -238,6 +243,11 @@ export async function genezioDeploy(options: GenezioDeployOptions) {
                 commandOptions: JSON.stringify(options),
             });
         }
+
+        // If name was provided but no matching frontend was found, throw an error
+        if (options.name && frontendUrls.length === 0) {
```

Comment:
This check is a bit misleading - it doesn't do what the comment says.

It just checks if there are any frontendUrls deployed.

Reply:
And imo the check is done pretty late - after iterating through all the frontends and attempting to deploy.

I think this check can be done outside the `for` loop - if `options.name` doesn't match any frontend.name in the yaml - we should give a warning that the `options.name` doesn't exists + check frontend.name and add them.